### PR TITLE
WCOR-0: Fix workflow issue.

### PR DESCRIPTION
**Pull request by gatoona**
Fix workflow issue where setting moderated state to unpublish/publish did not reflect correct publish state of node.

### CODE REVIEW

**Comment by gatoona** [comment]
@joshuami this PR fixes the workflow issue @jmilden-git brought up on today's sprint meeting.

File: config/default/workflows.workflow.editorial.yml [hunk 1]
```diff
@@ -31,14 +31,13 @@ type_settings:
       weight: -1
     unpublished:
       published: false
-      default_revision: false
+      default_revision: true
```
Comment (joshuami) [inline config/default/workflows.workflow.editorial.yml:34]: @gatoona, you beat me to it. I was just about to send a screenshot to the team chat that the issue was with the default revision setting 😄

File: config/default/workflows.workflow.editorial.yml [hunk 2]
```diff
@@ -31,14 +31,13 @@ type_settings:
...
       label: Unpublished
       weight: 1
   transitions:
     draft:
       label: 'Create new draft'
       from:
-        - draft
```
Comment (joshuami) [inline config/default/workflows.workflow.editorial.yml:41]: @gatoona, you should leave the settings that allow you to create a new revision in the same state as the current state. Creating 2-3 drafts before moving an edit to in review and have 2-3 in review comments or changes is pretty common. LIkewise, you'll want to be able to make an edit to published content and immediately publish that edit. 

Users get terribly confused when they don't see a moderation state for simple workflows like this. 😄

**Review by joshuami — CHANGES_REQUESTED** [review]

File: config/default/workflows.workflow.editorial.yml (hunk 2 above)
Comment (gatoona) [reply config/default/workflows.workflow.editorial.yml]: @joshuami good point. Added back in. Did keep the transition from draft -> unpublish unchecked.

File: config/default/workflows.workflow.editorial.yml [hunk 3]
```diff
@@ -65,7 +65,6 @@ type_settings:
     unpublish:
       label: Unpublish
       from:
-        - draft
```
Comment (joshuami) [inline config/default/workflows.workflow.editorial.yml:68]: This is probably fine. I can't think of a great use case for unpublishing from any state other than published. 😄

**Review by joshuami — APPROVED** [review]

**Review by lgbridges — COMMENTED** [review]
Is QA still needed?

**Comment by gatoona** [comment]
> Is QA still needed?

@lgbridges feel free to merge.

**Pull request merged**